aries-site/src/pages/learn/index.js

```diff
@@ -16,12 +25,73 @@ const Learn = () => (
       description={pageDetails.seoDescription}
       canonicalUrl="https://design-system.hpe.design/learn"
     />
-    <PageContent gap="large">
+    <PageContent>
       <Box pad={{ vertical: 'medium' }} justify="center" width="large">
         <Heading margin="none">{title}</Heading>
         <Paragraph size="large">{pageDetails.description}</Paragraph>
       </Box>
-      <CardGrid cards={cards} pad={{ bottom: 'large' }} headingLevel={2} />
+      <Data data={cards} pad={{ bottom: 'large' }}>
+        <DataSearch width={{ max: 'medium', width: '100%' }} />
```

Comment:
Wondering how we might communicate the categories earlier in the page flow? As a reader, I might be interested in knowing these before I start browsing.

`<DataFilter />` might be one option. Another might be exploration of a [toggle-button](https://mui.com/material-ui/react-toggle-button/) which would also solve for the GLCP use case where they are using cards to filter.

Doesn't need to be in this work. Just sharing thoughts.

Reply:
Totally agree. Given the size of typography and cards, you can't get a quick sense of what's available. I'll do a separate  design exploration on how to surface that and handle in a different PR once we have an idea of direction